example/basic/server.js

```diff
@@ -1,49 +1,49 @@
 'use strict';
+
 var Hapi = require('hapi');
 var Vision = require('vision');
-var assert = require('assert')
+var assert = require('assert');
 var HapiRiotViews = require('../../lib/index.js');
 
 var server = new Hapi.Server();
 var port = process.env.PORT || 8000;
+
 server.connection({ port: port });
-server.register(Vision, (err) => {
-  console.log(err);
+
+server.register(Vision, function (err) {
+  console.log(err); // eslint-disable-line
```

Comment:
please use `server.log` instead of `console.log(err); // eslint-disable-line` ...
see: https://github.com/dwyl/learn-hapi#logging-with-good